components/org.wso2.carbon.identity.governance/src/main/java/org/wso2/carbon/identity/governance/common/IdentityConnectorConfig.java

```diff
@@ -34,4 +38,9 @@
 
         return null;
     }
+
+    //meta method
```

Comment:
Since this is a public method, we need to add a proper method comment.

Reply:
Fixed in commit 8032899333352cf8a310623b49856dfb99ec9767